physchem_calc.py

```diff
@@ -51,14 +60,16 @@ def calc(smi, name):
             mw = rdMolDescriptors._CalcMolWt(m)
             csp3 = rdMolDescriptors.CalcFractionCSP3(m)
             hac = m.GetNumHeavyAtoms()
+            nstereo = GetStereoisomerCount(m)
```

Comment:
This returns number of isomers considering unassigned centers only that is not desirable.
The total number of chiral centers (assigned and unassigned) should be returned.
nstereo variable is better to name properly - n_chiral_centers

Reply:
3